rpc_jobs/release.yml

```diff
@@ -0,0 +1,97 @@
+- job:
+    name: RE-Release
+    project-type: workflow
+    concurrent: false
+    properties:
+      - build-discarder:
+          num-to-keep: 30
+    parameters:
+      - rpc_gating_params
+      - string:
+          name: RELEASE_VERSION
+          description: |
+            The version that will be tagged by this job
+      - string:
+          name: MAINLINE_BRANCH
+          description: The MAINLINE_BRANCH branch to work from
+      - string:
+          name: REPO_URL
+          description: |
+            URL of the working repo. Used to pull and push, so should be
+            git@github.com:owner/repo. Key for
+            https://github.com/rpc-jenkins-svc is used for pushing.
+      - string:
+          name: MAIL_TO
+          description: Recipient of notification email
+      - string:
+          name: MAIL_SUBJECT
+          description: Subject of notification email
+          default: auto
+      - string:
+          name: MAIL_BODY
+          description: Body of notification email
+          default: auto
+      - string:
+          name: STAGES
+          description: |
+            Items to run. Options:
+              Tag,
+              Update RC Branch,
+              Mail Notification
+          default: >-
+            Tag,
+            Update RC Branch
+    dsl: |
+      stage("Check Parameters"){
+        if (env.REPO_URL == "" || env.MAINLINE_BRANCH == ""){
+          throw new Exception(
+            "REPO_URL and MAINLINE_BRANCH parameters are required.")
+        }
+        if (!(REPO_URL ==~ /git@github.com:[\w-_]*\/[\w-_]*/)){
+          throw new Exception(
+            "Repo url should be of the form git@github.com:owner/repo")
+        }
+        print ("Parameter check successful.")
+      }
+      library "rpc-gating@${RPC_GATING_BRANCH}"
+      common.shared_slave(){
+        rc = "${MAINLINE_BRANCH}-rc"
+        stage("Configure Git"){
+          common.configure_git()
+        }
+        stage("Clone"){
+          common.clone_with_pr_refs(REPO_URL, MAINLINE_BRANCH)
+        }
+        common.conditionalStage(
+          stage_name: "Tag",
+          stage: {
+            release.tag(RELEASE_VERSION, rc)
+          })
```

Comment:
What about updating https://github.com/rcbops/rpc-openstack/releases? I believe that is still being done as part of the release process.

Reply:
That will be done, but not by this job. That will be part of the SHA bump / Dependency update job.